packages/fast-components-react-msft/src/utilities/index.ts

```diff
@@ -0,0 +1,4 @@
+const DisplayNamePrefix: string = "Msft";
+
+export default DisplayNamePrefix;
```

Comment:
Do we need this default export? I think just the named export is fine.

Reply:
removed